kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/KafkaServerExtension.java

```diff
@@ -67,16 +82,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class KafkaServerExtension implements KieServerExtension, DeploymentEventListener, Runnable {
+public class KafkaServerExtension implements KieServerExtension, DeploymentEventListener, ProcessEventListener,
+                                  Runnable {
 
     public static final String EXTENSION_NAME = "Kafka";
     static final String KAFKA_EXTENSION_PREFIX = "org.kie.server.jbpm-kafka.ext.";
     static final String TOPIC_PREFIX = KAFKA_EXTENSION_PREFIX + "topics.";
+    static final String SIGNALS_ENABLED = KAFKA_EXTENSION_PREFIX + "enableSignals";
```

Comment:
we need two properties here.
KAFKA_EXTENSION_PREFIX.signals.mapping=(NONE) -> (by default or AUTO)
KAFKA_EXTENSION_PREFIX.messages.mapping=AUTO -> (or NONE)

the idea would be that in the future we can say extends which signals we need to map

Reply:
done